### ISSUE TITLE
Multitouch Surface 5 Pro not working 

### ISSUE THREAD

**Issue by Djrush86** [sebanc/brunch#2031]
After install the actually Brunch and Chrome OS I can only touch with one finger, no multi touch. Is there any solution for that?

**Comment by tejasraman** [comment]
> After install the actually Brunch and Chrome OS I can only touch with one finger, no multi touch. Is there any solution for that?

Do you have ipts_touchscreen enabled in framework options? That's required for full IPTS multitouch on some Surface Pro devices (Surface Pro 4/5/6)

**Comment by Djrush86** [comment]
Yes I have. I am currently on a older brunch r113 and rammus build r13. All is working fine here. No problem.

I will install this week a newer build and send the dmesg file from it.

**Comment by tejasraman** [comment]
> Yes I have. I am currently on a older brunch r113 and rammus build r13. All is working fine here. No problem.
> 
> I will install this week a newer build and send the dmesg file from it. 

Thanks so much @Djrush86
